install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -119,6 +119,8 @@ type WorkspaceConfig struct {
 		// Deprecated
 		UsageReportBucketName string `json:"usageReportBucketName"`
 	} `json:"contentService"`
+
+	EnableProtectedSecrets bool `json:"enabledProtectedSecrets"`
```

Comment:
nit: is `enabled` a typo? Just because in the example you gave in the description it was `enable` and kinda took me a minute to realize. :)

Reply:
Yes, this is a typo. Great catch :)